test/benchmark-larger/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_executable(realm-benchmark-larger main.cpp)
```

Comment:
Should this be built by default? If not, add `EXCLUDE_FROM_ALL`

Reply:
Good point